File: src/ProjectR.Scripting/content/scripts/spells/lua/LeaveCutter.lua
function GetTargetType()
    return Single
end

function GetName()
return "Leavecutter"
end

function GetDescription()
    return "Send sharp leaves flying at the enemy.\nVery large MR piercing."
end

function IsSupportSpell()
    return false
end

function GetMPCost()
    return .2
end

function GetDelay()
  return .5
end

function GetSpellType()
  return Magical
end

function GetMasteries()
  return { WND }
end

function SpellEffect()
  local damage = (3.375 * aMD * (aWND/100) - .15 * dMR) * (100/dWND)
  d:TakeDamage(damage)
end

File: src/ProjectR.Scripting/content/scripts/spells/LeaveCutter.cs
using System.Collections.Generic;
using ProjectR.Interfaces;
using ProjectR.Interfaces.Model;
using ProjectR.Interfaces.Model.Stats;

namespace ProjectR.Scripting.Spells
{
    public class LeaveCutter : SpellScriptBase
    {
        public override string Name { get { return "Leavecutter"; } }
        public override string Description { get { return "Send sharp leaves flying at the enemy.\nVery large MR piercing."; } }

        public override TargetType TargetType { get { return TargetType.Single; } }
        public override IList<EleMastery> Masteries { get { return new EleMastery[] { EleMastery.WND,  }; } }
        public override SpellType SpellType { get { return SpellType.Magical; } }
        public override bool IsSupportSpell { get { return false; } }

        public override double MPCost { get { return 20; } }
        public override double Delay { get { return .5; } }

        public override void SpellEffect(ICharacter caster, ICharacter target)
        {
            var damage = (3.375 * cMD * (cWND / 100) - .15 * tMR) * (100 / tWND);
            DealDamage(damage);
        }
    }
}